cachix-api/src/Cachix/Types/SwaggerOrphans.hs

```diff
@@ -1,31 +1,23 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 {-# LANGUAGE DataKinds #-}
-{-# LANGUAGE CPP #-}
 {-# LANGUAGE TypeOperators #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 module Cachix.Types.SwaggerOrphans
-  () where
+  where
 
-import Data.Proxy
-import Data.Swagger (ToParamSchema(..))
-import Servant.API
-import Servant.Swagger
-import Servant.Streaming
-import Servant.Auth.Swagger ()
-import Web.Cookie           (SetCookie)
+import Control.DeepSeq (NFData)
+import Data.Proxy (Proxy(..))
+import Data.Conduit (ConduitT)
+import Data.Swagger (ToParamSchema(..), ToSchema(..))
+import Servant.API  (NoContent)
 
-#if MIN_VERSION_swagger2(2,3,1)
-#else
-instance ToParamSchema SetCookie where
-  toParamSchema _ = mempty
-#endif
 
--- https://github.com/plow-technologies/servant-streaming/blob/master/servant-streaming-docs/src/Servant/Streaming/Docs/Internal.hs
--- TODO: these should define the body/response content
-instance (HasSwagger api) => HasSwagger (StreamBodyMonad contentTypes m :> api) where
-  toSwagger _ = toSwagger (Proxy :: Proxy api)
+-- TODO: upstream to servant-conduit
+instance ToSchema i => ToSchema (ConduitT i o m r) where
+  declareNamedSchema _ = declareNamedSchema (Proxy :: Proxy i)
+  -- TODO: Proxy o
```

Comment:
Looks like a nasty mismatch if you want to have a schema for the output. Does upstream have an issue about this?